addons/web/static/src/legacy/js/control_panel/search_utils.js

```diff
@@ -100,8 +101,8 @@ odoo.define('web.searchUtils', function (require) {
             id: 'this_month', groupNumber: 1, format: 'MMMM',
             addParam: {}, granularity: 'month',
         },
-        last_month: {
-            id: 'last_month', groupNumber: 1, format: 'MMMM',
+        previous_month: {
```

Comment:
Why is this getting renamed?

Reply:
I renamed these to try and prevent confusion with the newly added Last Month and Last Year Filters. The new filters have been given the name and id of `last_month` and `last_year` respectively.

Reply:
When patching core Odoo, I generally try to keep the patches as tight as possible. This seems brittle. Maybe just prefix all the new filters with mw_ and revert this rename?

Reply:
@amh-mw I've added a commit reverting the core name changes and prefixing the custom options with `mw_`. Let me know when you would like this rebased down to a single commit.